arithmetization/src/main/java/net/consensys/linea/zktracer/module/romlex/RomLex.java

```diff
@@ -238,9 +239,23 @@ public void callRomLex(final MessageFrame frame) {
                   }
                 });
       }
+
+      default -> throw new RuntimeException(
+          String.format("%s does not trigger the creation of ROM_LEX", getOpCode(frame)));
     }
   }
 
+  public void callRomLexForSelfdestruct(final Address addressWhichWillSelfdestruct) {
```

Comment:
I don't understand this, ROM / ROM_LEX shouldn't store operations with EMPTY bytecode

Reply:
Yes, we fixed this.